install/fenrus-install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: Scorpoon
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/revenz/Fenrus
+
+
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt install -y curl
+$STD apt install -y git
+msg_ok "Installed Dependencies"
+
+msg_info "Installing ASP.NET Core 7 SDK"
+var_os=$(grep "^ID=" /etc/os-release | cut -d'=' -f2 | tr -d '"')
+var_version=$(grep "^VERSION_ID=" /etc/os-release | cut -d'=' -f2 | tr -d '"')
+if [ "${var_os}" = "debian" ]; then
+  wget -q "https://packages.microsoft.com/config/debian/$var_version/packages-microsoft-prod.deb"
+  $STD dpkg -i packages-microsoft-prod.deb
+  rm packages-microsoft-prod.deb
+fi
+$STD apt-get update
+$STD apt-get install -y dotnet-sdk-7.0
+msg_ok "Installed ASP.NET Core 7 SDK"
+
+msg_info "Installing ${APPLICATION}"
+git clone https://github.com/revenz/Fenrus.git
+cd Fenrus || exit
+export DOTNET_CLI_TELEMETRY_OPTOUT=1
+dotnet publish -c Release -o "/opt/${APPLICATION}/" Fenrus.csproj
+gitVersionNumber=$(git rev-parse HEAD)
+echo "$gitVersionNumber" >"/opt/${APPLICATION}_version.txt"
+cp -r Apps/ "/opt/${APPLICATION}/"
```

Comment:
cp -r Apps/ "/opt/${APPLICATION}/" not needed

Reply:
Did you try it yourself? Without it the icons for the apps aren't shown and it's also done in the "Dockerfile" on the frenus repo.

Reply:
Ah hold on, was to fast. Just saw that you want the raw repo folder and the build output in the same place. I will have a look into it

Reply:
![Screenshot 2024-05-05 8 42 30 AM](https://github.com/tteck/Proxmox/assets/21342670/69bb420b-d1e1-4f81-8659-6140fe2438d8)

Reply:
Yeah when they are merged it works, you are right ;)

I hope this won't bring up some problems later when the folders are the same.